perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -103,7 +103,7 @@ def _PushRobustCommandScripts(self):
                                             os.path.basename(f)))
         self._has_remote_command_script = True
 
-  def RobustRemoteCommand(self, command):
+  def RobustRemoteCommand(self, command, should_log=False):
```

Comment:
Depending on the use here you might also want to support a timeout like other remote commands.